### PR TITLE
Remove graphql, remove fiat currency selector

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,7 +2,6 @@
 ESLINT_NO_DEV_ERRORS=true
 REACT_APP_AMPLITUDE_PROXY_URL="https://api.uniswap.org/v1/amplitude-proxy"
```
Comment (MicahZoltu) [inline .env:3]: We should figure out what uses this and remove it.  This is their tracking stuff.  They even route tracking through a proxy so people cannot evade it with tracker blockers.  😡

File: src/types/types-and-hooks.ts
Comment (MicahZoltu) [inline src/types/types-and-hooks.ts:1]: Why is this showing up as a new file?  I'm guessing you didn't just write this up real quick...

**Review by MicahZoltu — APPROVED** [review]

File: .env (hunk 1 above)
Comment (KillariDev) [reply .env:3]: I just created pr about it

File: src/types/types-and-hooks.ts
Comment (KillariDev) [reply src/types/types-and-hooks.ts:1]: there's script `https://github.com/DarkFlorist/Horswap/pull/52/files#diff-8a141be6d1c53dfbf77b68bff658480ed0d276ffd88ccd09303e29db5f26c195` that fetches and generates this file from the subgraph. I picked only the types we need from that file and included them in the codebase.

**Pull request merged**